vita3k/kernel/src/load_self.cpp

```diff
@@ -633,6 +633,18 @@ SceUID load_self(KernelState &kernel, MemState &mem, const void *self, const std
                     memcpy(seg_ptr.get(mem), seg_bytes, seg_header.p_filesz);
                 }
 
+                for (auto &patch : patches) {
+                    if (seg_index == patch.seg) {
+                        for (int i = 0; i < patch.values.size(); i++) {
+                            auto value = patch.values[i];
+
+                            LOG_INFO("Applying patch at offset 0x{:08X} with value 0x{:08X}", patch.offset + (sizeof(value) * i), patch.values[i]);
+
+                            *(uint8_t *)(seg_ptr.get(mem) + patch.offset + (sizeof(value) * i)) = patch.values[i];
+                        }
```

Comment:
Why not to use `memcpy` instead of whole loop?

Reply:
Good catch, thanks!